Furball.Vixie.TestApplication/Tests/TestSourceRect.cs

```diff
@@ -20,7 +20,7 @@ public class TestSourceRect : GameComponent {
             base.Initialize();
         }
 
-        private float _rotation = 1f;
```

Comment:
It defaulted to 1 for a reason (forcing you to look at any issues with rotation+source rect), it should stay that way

Reply:
oh, i never knew there was a reason, i changed it to see it more clearer as my head is bad at visualizing things, will change back